qemu/tests/numa_memdev_options.py

```diff
@@ -4,21 +4,34 @@
 
 from virttest import error_context
 from virttest import utils_misc
+from virttest import env_process
 from virttest.staging import utils_memory
 from virttest.compat_52lts import decode_to_text
-
-
-def check_host_numa_node_amount(test):
-    """
-    Check host NUMA node amount
-
-    :param test: QEMU test object
-    """
-    host_numa_nodes = utils_memory.numa_nodes()
-    host_numa_nodes = len(host_numa_nodes)
-    if host_numa_nodes < 2:
+from virttest.test_setup import HugePageConfig
+
+
+def check_host_numa_node(test):
+    #     """
+    #     Check host NUMA node whose node size is not zero
+    #
+    #     :param test: QEMU test object
+    #     """
+    host_numa = utils_memory.numa_nodes()
+    node_list = []
+    if len(host_numa) < 2:
         test.cancel("The host numa nodes should be at least 2! But there is %d."
-                    % host_numa_nodes)
+                    % len(host_numa))
+    for i in range(len(host_numa)):
+        node_size = int(process.getoutput(
+            "numactl -H | grep -w 'node %s' | grep size"
+            % str(host_numa[i])).split()[-2])
```

Comment:
no need to convert `host_numa[i]` to string.

Reply:
I agree it, if no need to sort size value, it is indeed not necessary to convert